### DISCUSSION TITLE
Make prism ignore current theme

### DISCUSSION THREAD

**Discussion by JohnDoe453** [PrismJS/prism#3103] [Q&A]
I can't figure out how i can make prism to continue using the light theme colors even after changing theme. 
I am using gatsby.

`./babelrc.js`

```
 [
      'prismjs',
      {
        languages: [
          'clike',
          'css',
          'html',
          'javascript',
          'markup',
          'mathml',
          'python',
          'svg',
          'xml',
          'sql',
          'typescript'
        ],
        theme: 'One Light',
        css: true,
        plugins: ['line-numbers']
      }
]
```

**Comment by RunDevelopment** [comment]
One light is a theme from [Prism themes](https://github.com/PrismJS/prism-themes). The babel plugin only supports [the themes in the main repo](https://github.com/PrismJS/prism/tree/master/themes), you have to import themes from Prism themes manually.

**Reply by JohnDoe453** [reply]
Alright, suppose I make it default, then I don't want it to change the colors according to the theme. I want it to be light all the time.

here the issue, i am facing :-

Light Theme:
![image](https://user-images.githubusercontent.com/86872191/135715936-27563dde-1c29-439f-9b8b-fa81175c3ee1.png)

Dark Theme: 
![image](https://user-images.githubusercontent.com/86872191/135715956-b4064713-6786-4c68-bc8d-3aba5675ed4f.png)

I want same highlighting colour for dark theme too. Is there any way to do this?

**Reply by RunDevelopment** [reply]
So your problem is that the dark theme doesn't work?

Keep in mind that Prism only supports one theme at a time. If you need multiple themes (e.g. to switch between dark mode and light mode), then you must only import either the light theme or the dark theme. You must not load both at the same time. The easiest way to do this is by dynamically adding and removing `<style>` or `<link>` tags.
